docs/roles/frontend.md

```diff
@@ -1,48 +1,52 @@
-# Compass
-
-Install Compass and incidentally Sass via `apt-get`. The Ruby role is installed as dependency.
-
 # Gulp
 
+_Existing configuration files will not be overriden._
+
 * Install `gulp` globally in the Vagrant box
-* Create a prefilled `Gulpfile.js` (if non-existant) with useful tasks (BrowserSync proxy for live reloading, Sass compilation with Autoprefixer & source maps, images optimization)
-* Create the associated `package.json` (if non-existant) containing the necessary dependencies to be installed with NPM.
+* Create a prefilled `Gulpfile.js` with useful tasks
+    * Watch & live reload with BrowserSync
+    * Compile Sass with Autoprefixer & source-maps
+    * Bundle JavaScript with Webpack, preconfigured with Babel (optional)
+    * Lossless images optimization with ImageMin
+* Create associated `gulp.config.js` and `webpack.config.js`
+* Create the `package.json` containing the necessary dependencies to be installed with NPM.
 
-After the first provisioning, you should edit the `Gulpfile.js` (especially the config block at the top) to match your project structure.
+After the first provisioning, you should edit the `gulp.config.js` and `webpack.config.js` to match your project structure.
 
 ## Parameters
 
-* **gulp_directory**: where should the gulpfile be created, defaults to <project_dir>/
-* **gulp_package_json_path**: where should the package.json file be created, defaults to <project_dir>/package.json
-* **gulp_package_json_author**: Author that should be put in the package.json file, defaults to "Liip AG"
-* **gulp_package_json_readme**: Documentation file the should be put in the package.json file, defaults to README.md
-* **gulp_use_webpack**: Do you want to install WebPack alongside Gulp, defatuls to true
+* **gulp_directory**: where should the gulpfile be created, defaults to `<project_dir>/`
+* **gulp_package_json_path**: where should the package.json file be created, defaults to `<project_dir>/package.json`
+* **gulp_package_json_author**: Author that should be put in the package.json file, defaults to `Liip AG`
+* **gulp_use_webpack**: Setup Webpack alongside Gulp, defaults to `true`
+* **gulp_use_purescript**: Add PureScript support to Webpack, defaults to `false`
+* **gulp_browserslist**: Define [Browserslist](https://github.com/ai/browserslist) in `package.json`, defaults to `["Last 2 versions", "IE 11"]`
```

Comment:
It would be better if this would be a yml list instead of a string (and then either build the list in the `package.json` file, or just use [`{{ gulp_browserslist|tojson }}`](http://jinja.pocoo.org/docs/2.9/templates/#tojson) which might also work). Whatever you choose to do, document whether it's a list or a string because that's not clear.